bin/generate_cli_output

```diff
@@ -22,6 +22,7 @@ save_usage sawtooth admin keygen
 save_usage sawtooth batch
 save_usage sawtooth batch list
 save_usage sawtooth batch show
+save_usage sawtooth batch-status
```

Comment:
I missed the PR that added this sub-command.  As far as the documentation goes, I have no problem with it.  However, I started a discussion on RocketChat about whether or not this is the appropriate name for the sub-command.